src/app/store.ts

```diff
@@ -1,10 +1,16 @@
 import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
-import counterReducer from '../features/counter/counterSlice';
+import usersReduser from '../features/UsersAPI';
```

Comment:
Typographical error in variable name. It should be 'usersReducer' instead of 'usersReduser'.